server/src/main/java/io/crate/metadata/settings/CoordinatorSessionSettings.java

```diff
@@ -170,4 +174,16 @@ public void statementTimeout(TimeValue statementTimeout) {
     public void memoryLimit(int memoryLimit) {
         this.memoryLimit = memoryLimit;
     }
+
+    public Map<String, Object> toMap() {
```

Comment:
@mfussenegger can I please ask you about this?
One on hand, this keeps it close to the settings, on the other hand, with an UNDEFINED object, we cannot query like so: `select settings['search_path']` from sys.sessions`
So I'm thinking if instead we should define the object columns instead, like here for example: https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/metadata/sys/SysNodesTableInfo.java#L141.

What's your take?

Reply:
Doesn't `addDynamicObject` on the system table builder support the subscript case?

Reply:
yes, it does, thank you!